musttag.go

```diff
@@ -216,21 +213,22 @@ func (c *checker) checkStruct(styp *types.Struct, tag string) (valid bool) {
 		if !ok {
 			// tag is not required for embedded types; see issue #12.
 			if !field.Embedded() {
-				return false
+				pass.Reportf(pos, "the given struct should be annotated with the `%s` tag: %s.%s (%s)",
```

Comment:
Moving this here would cause the same call to be reported multiple times. We don't want that.